src/packages/tests/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -88,14 +80,19 @@ type Database<Client> = {
    * @remarks This is used as the default provider name for the Prisma schema datasource block.
    */
   name: string
+  /** 
+   * Supply the enabled preview features for the prisma client. 
+   * 
+   */
+  previewFeatures?: string[] 
```

Comment:
Relevant for sqlserver feature, added helper function to not rely on the implementation detail

Reply:
Good idea!